RecoPixelVertexing/Configuration/python/customizePixelTracksSoAonCPU.py

```diff
@@ -0,0 +1,61 @@
+import FWCore.ParameterSet.Config as cms
+
+def customizePixelTracksSoAonCPU(process):
+  
+  process.CUDAService = cms.Service('CUDAService',
+    enabled = cms.untracked.bool(False)
+  )
+
+  # ensure the same results when running on GPU (which supports only the 'HLT' payload) and CPU
+  process.siPixelClustersPreSplitting.cpu.payloadType = cms.string('HLT')
+
+  from RecoLocalTracker.SiPixelRecHits.siPixelRecHitSoAFromLegacy_cfi import siPixelRecHitSoAFromLegacy
+  process.siPixelRecHitsPreSplitting = siPixelRecHitSoAFromLegacy.clone(
+    convertToLegacy = True
+  )
+
+  from RecoPixelVertexing.PixelTriplets.caHitNtupletCUDA_cfi import caHitNtupletCUDA
+  process.pixelTrackSoA = caHitNtupletCUDA.clone(
+    onGPU = False,
+    pixelRecHitSrc = 'siPixelRecHitsPreSplitting'
+  )
+
+  from RecoPixelVertexing.PixelVertexFinding.pixelVertexCUDA_cfi import pixelVertexCUDA
+  process.pixelVertexSoA = pixelVertexCUDA.clone(
+    onGPU = False,
+    pixelTrackSrc = 'pixelTrackSoA'
+  )
+
+  from RecoPixelVertexing.PixelTrackFitting.pixelTrackProducerFromSoA_cfi import pixelTrackProducerFromSoA
+  process.pixelTracks = pixelTrackProducerFromSoA.clone(
+    pixelRecHitLegacySrc = 'siPixelRecHitsPreSplitting'
+  )
+
+  from RecoPixelVertexing.PixelVertexFinding.pixelVertexFromSoA_cfi import pixelVertexFromSoA
+  process.pixelVertices = pixelVertexFromSoA.clone()
+
+  process.reconstruction_step += process.siPixelRecHitsPreSplitting + process.pixelTrackSoA + process.pixelVertexSoA
```

Comment:
```suggestion
  from RecoPixelVertexing.PixelTrackFitting.pixelTrackProducerFromSoA_cfi import pixelTrackProducerFromSoA
  process.pixelTracks = pixelTrackProducerFromSoA.clone(
    pixelRecHitLegacySrc = 'siPixelRecHitsPreSplitting'
  )

  process.reconstruction_step += process.siPixelRecHitsPreSplitting + process.pixelTrackSoA
```

Reply:
this should let it build the legacy pixel vertices on the new pixel tracks; we should re-enable the new pixel vertices after the net PR.